docs/reference/security.md

```diff
@@ -111,70 +111,20 @@ Here's an example .vsls.json file:
 ```json
 {
     "$schema": "http://json.schemastore.org/vsls",
-    "gitignore":"none",
     "excludeFiles":[
         "*.p12",
         "*.cer",
         "token",
         ".gitignore"
-    ],
-    "hideFiles": [
-        "bin",
-        "obj"
-    ]
+    ] 
 }
 ```
 
 > [!NOTE]
 > You can also make the all files/folders you share **read-only** when you start a collaboration session. See [below](#read-only-mode) for details.
 
-Let's walk through how these properties change what guests can do.
-
-### Properties
-
 The **excludeFiles** property allows you to specify a list of glob file patterns (very much like those found .gitignore files) that prevents Live Share from opening certain files or folders for guests. Be aware that this is inclusive of scenarios like a guest _following or jumping to your edit location, stepping into a file during collaborative debugging, any code navigation features like go to definition, and more._ It is intended for files you never want to share under any circumstances like those containing secrets, certificates, or passwords. For example, since they control security, .vsls.json files are always excluded.
 
-The **hideFiles** property is similar, but not quite as strict. These files are simply hidden from the file tree. For example, if you happened to step into one of these files during debugging, it is still opened in the editor. This property is primarily useful if you do not have a .gitignore file setup (as would be the case if you are using a different source control system) or if you simply want to augment what is already there to avoid clutter or confusion.
-
-The **gitignore** setting establishes how Live Share should process the contents of .gitignore files in shared folders. By default, any globs found in .gitignore files are treated as if they were specified in the "hideFiles" property. However, you can choose a different behavior using one of the following values:
-
-| Option    | Result                                                                                                                 |
```

Comment:
Can we keep this grid because its a good visual, and also the example below helps. Lets update it instead.

Reply:
The grid is explaining what the options are for the .gitignore property. There is no support for any of these options so I don't think we can keep the table.